tests/common/file_utils.rs

```diff
@@ -58,3 +63,10 @@ pub fn create_file_with_content(path: &PathBuf, content: &str) -> () {
     file.write_all(content.as_bytes())
         .expect(&format!("cannot write to file {:?}", path));
 }
+
+pub fn read_file(path: &PathBuf) -> String {
+    let mut file = File::create(&path).unwrap();
```

Comment:
calling [`File::create`](https://doc.rust-lang.org/stable/std/fs/struct.File.html#method.create) will create the file if it does not exist or will truncate the file if it already exists, is it the original intent ?

Reply:
Yes this is mistake, removing method